www/src/assets/plugins-ecosystem.svg

```diff
@@ -0,0 +1,38 @@
+<svg  viewBox="0 0 32 32" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
nit: but we can/should optimize/prettify these. I've always liked https://jakearchibald.github.io/svgomg/

Reply:
sure